#goal
float32 x_position_to_waypoint
float32 y_position_to_waypoint
---
#result
duration total_time
---
#feedback
float32 x_distance_to_goal
float32 y_distance_to_goal